src/repositories/auth.ts

```diff
@@ -0,0 +1,9 @@
+import { OAuth1Client } from '@extrahorizon/javascript-sdk';
```

Comment:
This file feels really weirdly named 😂 We call it `repository/auth` but there's nothing `sdk.auth` related in it.

I kinda get the getHost thing being here, but shouldn't the `.me` be in `repository/user`

Reply:
Fair and understandable. I was kinda looking at it from the perspective of the CLI itself, rather then the SDK. Not sure if that makes sense?